htdocs/modules/system/class/maintenance.php

```
@@ -166,12 +172,58 @@ public function CleanAvatar()
+            //     so 'avatarsX/...' doesn't satisfy 'avatars')
+            //   - confirm it's a regular file before removal.
+            // Only then invoke the cleanup helper.
+            // (int) cast on avatar_id is defence-in-depth: the value is
+            // DB-origin so SQL injection is implausible, but the project
+            // convention is to never concatenate non-cast values into
+            // SQL strings. The cast also silences SonarCloud's
+            // concatenation warning on these DELETE statements.
+            $avatarId   = (int) ($myrow['avatar_id'] ?? 0);
+            $avatarFile = ltrim(str_replace('\\', '/', (string) ($myrow['avatar_file'] ?? '')), '/');
+            if ('' === $avatarFile) {
+                $result1 = $this->db->exec('DELETE FROM ' . $this->db->prefix('avatar') . ' WHERE avatar_id=' . $avatarId);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$result1'. (undefined)

(UnusedLocalVariable)